pkg/hubble/relay/observer/server.go

```diff
@@ -274,10 +282,11 @@ func (s *Server) ServerStatus(ctx context.Context, req *observerpb.ServerStatusR
 			client := s.opts.ocb.observerClient(&p)
 			status, err := client.ServerStatus(ctx, req)
 			if err != nil {
-				s.opts.log.WithFields(logrus.Fields{
-					"error": err,
-					"peer":  p,
-				}).Warning("Failed to retrieve server status")
+				s.opts.log.Warn(
+					"Failed to retrieve server status",
+					logfields.Error, err,
+					logfields.Peer, p,
```

Comment:
ditto question about
```suggestion
					logfields.Peer, p.Name,
```

Reply:
Same